eng/Npm.Workspace.nodeproj

```diff
@@ -36,9 +56,19 @@
   </Target>
 
   <Target Name="Pack">
+    <PropertyGroup>
+      <PackageVersion>$(VersionPrefix)$(VersionSuffix)</PackageVersion>
+    </PropertyGroup>
     <Message Text="Packing NPM packages..." Importance="high" />
     <MakeDir Directories="$(PackageOutputPath)" Condition="!Exists('$(PackageOutputPath)')" />
-    <Exec Command="node $(MSBuildThisFileDirectory)scripts\pack-workspace.js $(RepoRoot)package.json $(PackageVersion) $(PackageOutputPath)" />
+    <Exec Command="node $(MSBuildThisFileDirectory)scripts\npm\pack-workspace.mjs --create-packages $(RepoRoot)package.json $(PackageVersion) $(PackageOutputPath) $(IntermediateOutputPath)" />
+    <ItemGroup>
+      <_NpmGeneratedPackages Include="$(PackageOutputPath)\*.tgz" />
+    </ItemGroup>
+    <PropertyGroup>
+      <_NpmFiles>@(_NpmGeneratedPackages->'📦 %(FileName)%(Extension)', '%0a%09')</_NpmFiles>
+    </PropertyGroup>
+    <Message Importance="High" Text="$(MSBuildProjectName) -> %0a%09$(_NpmFiles)" />
```

Comment:
This prints the outputs to the terminal logger
![image](https://github.com/dotnet/aspnetcore/assets/6995051/7388eb12-9cca-49b1-9964-0725f99e245a)